template/README.md

```diff
@@ -0,0 +1,22 @@
+This guide provides in the broad strokes a set of bash best practices that are expected to be adhered to.
+
+### Best practices and tips
+
+- Include a one-line comment description at the top of the script explaining what the script does.
+- Always double quote variables.
```

Comment:
Does the mean when de-referencing, always use double quotes?  eg, "$variable".
This may not apply if using that in a for loop.
list="a b c"
for i in "$list" //<-- the ShellCheck will prompt an error, because the for loop may take it as one single item.

Reply:
Ah right, might need to reword that to something like:
```suggestion
- Always deference variables with double quotes. Ex: `"$variable"`
```

Reply:
It is expected that whoever uses managed scripts know not to dereference the iterator var over loops :)